gwtp-core/gwtp-crawler-guice/src/main/java/com/gwtplatform/crawler/server/guice/ServiceKey.java

```diff
@@ -14,12 +14,13 @@
  * the License.
  */
 
-package com.gwtplatform.crawler.server;
+package com.gwtplatform.crawler.server.guice;
```

Comment:
That's a breaking change. We should either use inheritance or duplicate it and support classes from both the new and old packages.